### PR TITLE
feat(connector): [FISERVEMEA] Add template code

### PR DESCRIPTION
**Pull request by deepanshu-iiitu**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Template code added for new connector Fiserv-EMEA
https://docs.fiserv.dev/

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context

https://github.com/juspay/hyperswitch/issues/5582

## How did you test it?

Only template code added hence no testing required.

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by swangi-kumari — DISMISSED** [review]

**Review by swangi-kumari — APPROVED** [review]

File: scripts/add_connector.sh
Comment (SanchithHegde) [inline scripts/add_connector.sh:1]: We can move this file under connector team code ownership moving forward.

**Review by sai-harsha-vardhan — APPROVED** [review]

File: crates/router/tests/connectors/fiservemea.rs
Comment (SanchithHegde) [inline crates/router/tests/connectors/fiservemea.rs:1]: These would have to be moved to the `hyperswitch_connectors` crate as well, we'd have to update the connector addition script. We can take this up in a separate PR, not a problem.

**Review by SanchithHegde — APPROVED** [review]

File: scripts/add_connector.sh
Comment (deepanshu-iiitu) [reply scripts/add_connector.sh:1]: Sure

File: crates/router/tests/connectors/fiservemea.rs
Comment (deepanshu-iiitu) [reply crates/router/tests/connectors/fiservemea.rs:1]: Ok

**Pull request merged**